.buildkite/scripts/set-bk-analytics-token.sh

```diff
@@ -0,0 +1,4 @@
+# !/bin/bash
+echo "--- Set BUILDKITE_ANALYTICS_TOKEN :vault:"
```

Comment:
should we define the shebang here at least the pipefail so if this fails, it would stop the execution